src/modules/whitelist/main_impl.h

```diff
@@ -153,6 +157,7 @@ int secp256k1_whitelist_signature_parse(const secp256k1_context* ctx, secp256k1_
 int secp256k1_whitelist_signature_serialize(const secp256k1_context* ctx, unsigned char *output, size_t *output_len, const secp256k1_whitelist_signature *sig) {
     VERIFY_CHECK(ctx != NULL);
     ARG_CHECK(output != NULL);
+    ARG_CHECK(output_len != NULL);
```

Comment:
In include/secp256k1_whitelist.h can you also add a `SECP256K1_NONNULL` flag to the function declaration?